projects/storefrontlib/src/cms-components/product/product-images/product-images.component.ts

```diff
@@ -15,7 +15,7 @@ export class ProductImagesComponent {
   private product$: Observable<Product> = this.currentProductService
     .getProduct()
     .pipe(
-      filter(Boolean),
+      filter(isNonNullable),
```

Comment:
In other cases for product service you used `isNotNull`. Can you help me understand when when we should use which?